recipe/meta.yaml

```diff
@@ -12,10 +14,9 @@ source:
     - skip-broken-tests.patch
 
 build:
-  number: 0
+  number: {{ build_number }}
   skip: true  # [py<37]
   # There isn't currently available tifffile >=2019.7.26 on win32 
-  skip: true  # [win32]
   entry_points:
     - skivi = skimage.scripts.skivi:main
 
```

Comment:
The package seems to require `numpy` version `1.19.2` due to some fixes that are required for `wheels`

https://github.com/scikit-image/scikit-image/blob/v0.19.3/pyproject.toml#L12&&L36

Reply:
Yeah, numpy in `host` can be modified. See an example for `scikit-learn` https://github.com/AnacondaRecipes/scikit-learn-feedstock/blob/21d76097066f953006d218ab94e252200bad8fcd/recipe/meta.yaml#L26-L27
numpy 1.17 and 1.18 hit EOL. So you can use 1.19 branch